.travis.yml

```diff
@@ -34,6 +36,23 @@ jobs:
         file:
           - stream.zip
           - stream-$TRAVIS_TAG.zip
+        on:
+          all_branches: true
+          repo: xwp/stream
+
+    - name: Tag Package
+      if: branch IS present AND type = push
+      script: npm run release
+      before_deploy:
+        - openssl aes-256-cbc -K $encrypted_49634e5b1863_key -iv $encrypted_49634e5b1863_iv -in ./local/travis/travis_deploy_key.enc -out ~/.ssh/id_rsa -d
```

Comment:
This is necessary because Travis [doesn't support custom SSH keys for public repos](https://docs.travis-ci.com/user/private-dependencies/#user-key) so we need to manually add a custom SSH key associated with the @team-atlas-bot machine user which has the necessary write permissions.